composer.json

```diff
@@ -33,7 +33,6 @@
     "compat": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --standard=phpcompat.xml.dist --report=summary,source",
     "format": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf --report=summary,source",
     "lint": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source",
-    "lint:errors": "@lint -n",
-    "serve": "npx @wp-now/wp-now start --php=7.2 --port=8881"
```

Comment:
Is the serve command being intentionally removed here?

Reply:
yes, in favor of just using wp-env